include/algorithms/decision_forest/decision_forest_classification_model.h

```diff
@@ -121,6 +121,20 @@ class DAAL_EXPORT Model : public classifier::Model
     */
     virtual size_t getNumberOfClasses() const = 0;
 
+    /**
+    *  Perform Depth First Traversal of i-th tree
+    *  \param[in] iTree    Index of the tree to traverse
+    *  \param[in] visitor  This object gets notified when tree nodes are visited
+    */
+    virtual void traverseDFS(size_t iTree, tree_utils::classification::interface1::TreeNodeVisitor& visitor) const = 0;
```

Comment:
I think it would be better (and it`s right way, as tests for **DF will call new function** as we have already seen) to place here new traversDFS and traverseBFS to keep old placement in VTable.
Move it on 103 and 110 lines.
Note: new functions can be without explicit interface.